singer-commons/src/main/thrift/loggingaudit_config.thrift

```diff
@@ -56,6 +56,11 @@ struct AuditConfig{
     *  true at Merced / Merced_HR stage.
     */
     3: optional bool stopAtCurrentStage = false;
+
+    /**
+    *   flag indicates whether to skip corrupted messages at the current stage or not
+    */
+    4: optional bool skipCorruptedMessageAtCurrentStage = false;
```

Comment:
since this is a new field introduced in AuditConfig,  at Singer side,  we need to parse the configuration if it is provided.  pls take a look at the method parseAuditConfig() in ConfigUtils.java